src/lib/Components/DottedLine.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from "react"
+import { requireNativeComponent, View } from "react-native"
+
+class DottedLine extends React.Component<any, any> {
```

Comment:
Can you add a TypeScript interface for the props that describes the `dotScale` prop? And specify that the state will be `null`?

Reply:
( there's some docs on how that works here: https://github.com/artsy/reaction/blob/master/docs/getting_started.md )

Reply:
yep, thank you!